src/backend/joanie/settings.py

```diff
@@ -544,6 +544,9 @@ class Test(Base):
             },
         }
     )
+    PASSWORD_HASHERS = [
+        "django.contrib.auth.hashers.MD5PasswordHasher",
+    ]
```

Comment:
I don't think this diff is related to the scope of this PR. But I'm curious to know what is the purpose to override this setting if this something deliberate.

Reply:
You're right, I will put it in another commit, and change the PR title.
Using this cuts a minute in the tests running time.
It's used in generate_password, and called many times in our tests.